crates/sequencer/src/rpc.rs

```diff
@@ -92,5 +95,39 @@ pub(crate) fn create_rpc_module<C: sov_modules_api::Context>(
             },
         }
     })?;
+
+    rpc.register_async_method(
+        "citrea_sendRawDepositTransaction",
+        |parameters, ctx| async move {
+            let mut params = parameters.sequence();
+            let deposit: Bytes = params.next().unwrap();
+
+            info!("Sequencer: citrea_sendRawDepositTransaction");
+
+            let evm = Evm::<C>::default();
+            let mut working_set = WorkingSet::<C>::new(ctx.storage.clone());
+
+            let dep_tx = ctx
+                .deposit_mempool
+                .lock()
+                .await
+                .make_deposit_tx_from_data(deposit.clone().into());
+
+            let tx_res = evm.get_call(dep_tx, None, None, None, &mut working_set);
```

Comment:
Why do we need to check it? Please add comment

Reply:
so basically our bridge backend is going to send calls to the bridge contract to our sequencer to be run as system transactions. we want to check if its not working in order not to include a failing tx in our soft confirmations.